snapshottest/module.py

```diff
@@ -236,9 +238,17 @@ def assert_match(self, value, name=''):
     def save_changes(self):
         self.module.save()
 
+    @classmethod
+    def remove_fields_from_dict(cls, dictionary, remove_fields=None):
+        if remove_fields is None:
+            remove_fields = []
+        for field in remove_fields:
```

Comment:
This won't work with nested fields, right?
something like
```json
{
    "content": {
        "created_at": "......."
    }
}
```

Reply:
You're right, in this case it will not work. What do you suggest in this case since we can have a situation like this:
```
{
    "created_at": ... ,
    "content": {
        "created_at": "......."
    }
}
```
Should the remove_fields parameter perhaps be a list if we would want to separate these cases? In this example if you wanted to delete the non-nested created_at you could just pass 'created_at' or ['created_at'] and for the nested ['content', 'created_at'].

The second option would be to globally remove the given field, so if 'created_at' is given as the parameter, it should remove every appearance of it (including nested).

Reply:
Right. I'm not quite sure myself on what to do in this situation. One approach would be to specify not just keys but the whole path, kind of like "content.items.*.event.created_at" however, for my needs, I just forked this repo and made it delete every key that matches the name recursively.